Localize/LocProject.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
You shouldn't check in this file if it's being generated (which means you'll also have to add a new .gitignore file in this directory and add the name of this file to it).

Reply:
Okay, so we always want to add generated files to a .gitignore?

Reply:
Yes, because otherwise they'll show up when you do `git status`, and that's annoying.